scripts/bee.sh

```diff
@@ -86,6 +87,7 @@ PORT_MAPS=2
 SWAP=true
 SWAP_FACTORY_ADDRESS="0x5b1869D9A4C187F2EAa108f3062412ecf0526b24"
 INIT_ROOT_DATA_DIR="$MY_PATH/bee-data-dirs"
+PAYMENT_TOLERANCE="100000000000000000"
```

Comment:
this value is different than https://github.com/ethersphere/bee-factory/pull/21/files#diff-f3736fd0d9f6c379e7622c6230d742bb019c1120223e1c3857bb1808d27d5413R42 . Does it need to be defined in both places?